db/manager_test.go

```diff
@@ -172,12 +172,35 @@ func TestManager_GetDB(t *testing.T) {
 		_, err = collection.CreateMany([][]byte{person2, person3})
 		checkErr(t, err)
 
+		// Delete the db, we'll try to restart again
+		err = man.DeleteDB(ctx, id)
+		checkErr(t, err)
+
 		time.Sleep(time.Second)
 
 		err = man.Close()
 		checkErr(t, err)
 		err = n.Close()
 		checkErr(t, err)
+
+		t.Run("test get deleted db after restart", func(t *testing.T) {
```

Comment:
This is just double double checking that a db is actually deleted after deleting it.